src/main/java/com/basho/riak/client/api/commands/kv/FetchValue.java

```diff
@@ -75,8 +75,8 @@
 {
 
     private final Location location;
-    private final Map<Option<?>, Object> options =
-            new HashMap<Option<?>, Object>();
+    private final Map<RiakOption<?>, Object> options =
```

Comment:
TL;DR: It allows us to use either the FetchValue.Options or DeleteValue.options in the base MultiCommand class, for when we build the individual commands.  https://github.com/basho/riak-java-client/blob/788e0710a0d695701aba5bf63b229df657a7cbbe/src/main/java/com/basho/riak/client/api/commands/kv/MultiCommand.java#L194-L206

I also had to switch FetchValue and DeleteValue's options to use the shared superclass `RiakOption` over the local/subclass `Option` type, but this just lets us reuse the options on the individual commands without casting:
https://github.com/basho/riak-java-client/blob/788e0710a0d695701aba5bf63b229df657a7cbbe/src/main/java/com/basho/riak/client/api/commands/kv/MultiCommand.java#L70-L75

We can also then use it in the same way for the MultiCommand Builder: 
https://github.com/basho/riak-java-client/blob/788e0710a0d695701aba5bf63b229df657a7cbbe/src/main/java/com/basho/riak/client/api/commands/kv/MultiCommand.java#L132-L132

Yay reuse!

>Also, RiakOption is in the riak package, making the Riak prefix redundant. Why the change?
RiakOption's always been named as such, no changes there.